packages/publish/test/fixtures/config/valid-config.yml

```diff
@@ -0,0 +1 @@
+randomBoolean: true
```

Comment:
Update?

Reply:
:+1:  updated to reflect actual config format.